README.md

```diff
@@ -1 +1,118 @@
-# issac_ros2_control
+# Isaac Ros2 Control
+
+Description: This is a ROS 2 package for integrating the ros2_control with the [NVIDIA Isaac Sim](https://developer.nvidia.com/isaac-sim).
+
+
+<img src="https://picknik.ai/assets/images/logo.jpg" width="100">
+
+[![Build Status](https://github.com/PickNikRobotics/isaac_ros2_control/actions/workflows/build_and_test.yaml/badge.svg)](https://github.com/PickNikRobotics/isaac_ros2_control/actions/workflows/build_and_test.yaml)
+
+## Installation
+
+### Build from Source
+
+These instructions assume you are running on Ubuntu 20.04:
```

Comment:
Suggest adding a note that Isaac currently requires 20.04 but this repo it is recommended you run a 22.04 Docker image on 20.04 to be compatible with Humble.